astropy/uncertainty/distributions.py

```diff
@@ -44,8 +44,10 @@ def normal(center, *, std=None, var=None, ivar=None, n_samples,
 
     Returns
     -------
-    distr : ``cls``, usually `Distribution`
+    distr : `Distribution` or object
```

Comment:
Should this be `~astropy.uncertainty.Distribution`?

Reply:
yes, absolute paths are better, but both work. problems can arise when subclassing something with relative paths.

Reply:
✅